app/controllers/posts_controller.rb

```diff
@@ -30,4 +30,9 @@ def create
       end
     end
   end
```

Comment:
- [OPTIONAL] Please use CanCanCan for Post authorization in Post Controller. Something like so: 

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Please kindly that.